reference/pipeline/stages/index.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: single
+title:  "Pipeline Stages"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+> Note: this is a first pass at this documentation; more details will be added
+> soon.
+
+This article describes the currently-supported stages that you can add to your
+Spinnaker pipelines.
+
+## General
+
+### Bake
+Bakes an image in the specified region.
+
+### Canary Analysis
+Runs automated canary analysis against the deployment before fully deploying.
+For more information on automated canary analysis in Spinnaker, see the
+[user guide](/guides/user/canary/).
+
+### Check Preconditions
+Checks for preconditions before continuing. Specifically, you can either check
+that your cluster is a particular size, or add a pipeline expression. See the
```

Comment:
If these are the only two examples, and will probably remain so, then "Specifically..." is ok. But otherwise I would change this to "For example, you can..."